orchestrator-api/src/main/java/com/sequenceiq/cloudbreak/orchestrator/rotation/ServiceConfigRotationContext.java

```diff
@@ -1,6 +1,6 @@
 package com.sequenceiq.cloudbreak.orchestrator.rotation;
 
-import com.sequenceiq.cloudbreak.rotation.secret.RotationContext;
+import com.sequenceiq.cloudbreak.rotation.common.RotationContext;
 
 public abstract class ServiceConfigRotationContext extends RotationContext {
```

Comment:
as discussed, I wanted to avoid including flow and all heavy stuff for every module using some part of rotation code, thus I decided to separate heavy part of the framework (flow, db stuff, etc) and lightweight stuff like context classes, etc.

Reply:
As discussed, secret rotation related code dependencies have been removed from orchestrator-salt module, thus secret-rotation-common has been deleted.